gst-libs/mfx/gstmfxcontext.c

```diff
@@ -92,7 +92,7 @@ gst_mfx_context_replace (GstMfxContext ** old_context_ptr,
 	  GST_OBJECT(new_context));
 }
 
```

Comment:
Yep, lame attempt by me at a class base pointer. Maybe we should use a union, or just use compile-time macros. Or something like duck-typing?